dspace-server-webapp/src/test/java/org/dspace/app/rest/TiltedRelationshipRestRepositoryIT.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest;
+
+import static org.dspace.app.rest.matcher.MetadataMatcher.matchMetadata;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.dspace.builder.RelationshipBuilder;
+import org.dspace.content.MetadataSchemaEnum;
+import org.dspace.content.RelationshipType;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * This class carries out the same test cases as {@link RelationshipRestRepositoryIT}.
+ * The only difference being that a RelationshipType is set on
+ * {@link RelationshipRestRepositoryIT#isAuthorOfPublicationRelationshipType}.
+ */
+public class TiltedRelationshipRestRepositoryIT extends RelationshipRestRepositoryIT {
+
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+
+        context.turnOffAuthorisationSystem();
+
+        isAuthorOfPublicationRelationshipType.setTilted(RelationshipType.Tilted.LEFT);
```

Comment:
Same here...it'd be nice to have a second test in this class which verifies that a `Tilted.RIGHT` relationship also works as expected.